docs/pages/admin-guides/management/guides/migrating-iam-ic-from-okta-to-teleport.mdx

```diff
@@ -0,0 +1,325 @@
+
+Teleport will not create or delete Account Assignments on AWS accounts outside
+of its allow-list.
+
+## Step 5: Migrate Account Assignments
+
+We're now at the midway point, and ready to migrate Account Assignments from the
+Okta-managed groups into new Teleport-managed Access Lists. To migrate groups,
+create a new Access List in Teleport (taking care not to use the same name as
+the existing Okta-managed Group) and create the appropriate memberships and 
+Account Assignments.
+
+Account Assignments can be created on an Access List by assigning it the Account
+Assignment roles created by the Identity Center integration, assigning it a 
+custom Teleport role that specifies a specific combination of access, or a
+combination of each.
+
+For more information, see the [Identity Center integration guide](./aws-iam-identity-center.mdx)
+
+## Step 6: Expanding Teleport integration scope
+
+Once you are satisfied with the way Teleport is handling the initial set of
+imported AWS resources, you can expand the scope of the Identity Center
+integration by editing the plugin import features. 
+
+
+### With `tctl edit`
+
+<Admonition type="warning">
+This currently involves manually editing the Identity Center's integration 
+resource using `tctl`, which is a dangerous operation. Please ensure you take a
+backup of the plugin resource so you can roll back if necessary.
+
+A guided editing workflow is currently under development.
+</Admonition>
+
+You can expand the scope of the Teleport Identity Center integration by editing
+the integration's plugin resource with `tctl`.
+
+```console
+./tctl edit plugins/aws-identity-center
+```
+
+The plugin resource is a YAML document that looks something like this:
+
+```YAML
+kind: plugin
+version: v1
+metadata:
+  labels:
+    teleport.dev/hosted-plugin: "true"
+  name: aws-identity-center
+spec:
+  Settings:
+    aws_ic:
+      # Account import filters. An absent or empty list of filters implies "manage all AWS accounts"
+      aws_accounts_filters:
+      - Include:
+          id: "637423191929"
+      - Include:
+          id: "730335414865"
+      - Include:
+          id: "058264527036"
+      - Include:
+          name_regex: ^Staging-.*$
+
+      # User provisioning filters. An absent or empty list of filters implies "provision all users to AWS"
+      user_sync_filters:
+      - labels:
+          teleport.dev/origin: okta
+
+      # Group import filters. See notes below. 
+      group_sync_filters:
+      - Include:
+          name_regex: '^Group #00\d+$'
+
+      access_list_default_owners:
+      - admin
+      arn: arn:aws:sso:::instance/ssoins-722326ecc902a06a
+      credentials_source: 2
+      integration_name: aws-identity-center
+      provisioning_spec:
+        base_url: https://scim.us-east-1.amazonaws.com/f3v9c6bc2ca-b104-4571-b669-f2eba522efe8/scim/v2
+      region: us-east-1
+```
+
+You can add or remove filters to the various filters. Once you save and quit the
```

Comment:
"filters to the various filters" reads strange

Reply:
And this is why proofreading exists.